app/Http/Controllers/CompetitionsController.php

```diff
@@ -7,7 +7,7 @@
 
 class CompetitionsController extends Controller
 {
-    private $validationRules = [
+    public static $validationRules = [
```

Comment:
yeah, it's more about the needing to reference validation for Competitions in the WaitingRoomController.... that's the part that feels weird.